autogpt_platform/backend/backend/notifications/gist.py

```
@@ -0,0 +1,128 @@
+"""
+
+import re
+from typing import Any
+
+GIST_MAX_CHARS = 150
+
+# Output names that read as files rather than prose, so they are named by kind
+# and count instead of being quoted.
+# Whole words only: "profile" contains "file" and "document" contains "doc",
+# and either would route a text output into the file-describing branch.
+_FILE_HINT = re.compile(r"\b(file|image|video|audio|clip|pdf|doc|csv|sheet)s?\b", re.I)
```

Ruff (0.16.2)

[error] Use of regular expression alias `re.I`

Replace with `re.IGNORECASE`

(FURB167)